specs/mgt-search-suggestion.md

```diff
@@ -0,0 +1,161 @@
+# mgt-search-suggestion
```

Comment:
Wondering if this component could be called simply `mgt-search` or `mgt-search-box`? It functions like a search box that has suggestions.

Reply:
This component based on Search Suggestion API, suggestion result can be used for searching, so I guess mgt-suggestion
maybe better?

Reply:
let's use mgt-search

Reply:
Resolved